test/stdlib/MultilineErrors.swift

```diff
@@ -0,0 +1,63 @@
+// RUN: %target-build-swift %s 2> %t.warnings.txt || echo -n
```

Comment:
Please move the two tests from `test/stdlib` to `test/Parser`.  Change the "MultilineErrors" test to use `%target-typecheck-verify-swift` and `// extected-error`, like this:

```
// RUN: %target-typecheck-verify-swift

_ = """
    Eleven
  Mu
    """ // expected-error@-1{{invalid mix of multi-line string literal indentation}}
```